testSvgForPlotters.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/python
```

Comment:
Rather than this file, could we get a proper test?

Reply:
I would love to, if only I knew what you mean.

Reply:
Sure, there are plenty of example tests already that you can base a new test off in `qrcode/tests`

Install your checkout of the project locally (`pip install -e .[dev]`, usually done within an activated virtual environment) and you can run `pytest` to quickly execute all the tests.